src/tests/ftest/util/test_utils_container.py

```diff
@@ -383,23 +382,23 @@ def create_snap(self, pool=None, cont=None, snap_name=None, epoch=None,
             sys_name (str, optional): system name
         """
         self.log.info("Creating Snapshot for Container: %s", self.uuid)
-        if not pool:
-            pool = self.pool
-        if not cont:
-            cont = self.uuid
-        if not svc:
-            svc = ",".join(str(rank) for rank in self.pool.svc_ranks)
-        if not sys_name:
-            sys_name = self.pool.name.value
+#        if not pool:
+#            pool = self.pool
+#        if not cont:
+#            cont = self.uuid
+#        if not svc:
+#            svc = ",".join(str(rank) for rank in self.pool.svc_ranks)
+#        if not sys_name:
+#            sys_name = self.pool.name.value
```

Comment:
Is there any reason for keeping these lines?

Reply:
Ah got pushed accidentally with commented lines. Thanks for catching it. Should be resolved now.